po/meson.build

```diff
@@ -1,19 +1,13 @@
 gettext_deps = []
+dep_intl = []
 if get_option('nls')
-  if host_machine.system() == 'linux' or host_machine.system() == 'gnu'
-    gettext_deps = []
-  elif host_machine.system() == 'freebsd' or host_machine.system() == 'openbsd' or host_machine.system() == 'dragonfly'
-    gettext_deps = [cc.find_library('intl', has_headers: ['libintl.h'], dirs: ['/usr/local/lib'], header_args: inc_args)]
-  elif host_machine.system() == 'netbsd'
-    gettext_deps = [cc.find_library('intl', has_headers: ['libintl.h'], dirs: ['/usr/pkg/lib'], header_args: inc_args)]
-  else # darwin, cygwin
-    gettext_deps = [cc.find_library('intl', has_headers: ['libintl.h'])]
+  dep_intl = dependency('intl')
+  if dep_intl.found()
```

Comment:
Seems pretty pointless to check here unless you're going to mark this dependency as `required: false`.

Reply:
@eli-schwartz Thanks... I'm fixing that. The problem I'm having is on FreeBSD (and OpenBSD), libintl is found, but the build fails because libintl.h can't be found.

Reply:
> Seems pretty pointless to check here unless you're going to mark this dependency as `required: false`.

@eli-schwartz @mesonbuild docs need a little change:

```
intl_dep = dependency('intl')

if intl_dep.found()
  # build options that need gettext
  conf.set('ENABLE_NLS', 1)
endif
```

https://mesonbuild.com/Release-notes-for-0-59-0.html

Reply:
Yeah sorry, that's dumb. :) Not sure how I messed that one up.

I'll fix it later today.

Reply:
That's just how the cookie rolls sometimes.